Makefile

```diff
@@ -0,0 +1,8 @@
+help: ## Prints this help.
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+
+build: ## Builds the docker container and installs dependencies
+	docker build -t php7boilerplate .
```

Comment:
I'd suggest using another name for the image, something that reflects that you're using it for the purpose of this kata.